builder/nutanix/driver.go

```diff
@@ -731,6 +731,14 @@ func (d *NutanixDriver) CreateImageURL(ctx context.Context, disk VmDisk, vm VmCo
 	}
 	req.Spec.Resources.SourceURI = &disk.SourceImageURI
 
+	if disk.SourceImageChecksum != "" {
+
+		req.Spec.Resources.Checksum = &v3.Checksum{
+			ChecksumValue:     &disk.SourceImageChecksum,
+			ChecksumAlgorithm: StringPtr("SHA_256"),
```

Comment:
Why is SHA_256 hardcoded?

Reply:
because for now decision was take to expose only the SHA_256 format and nont compelxify the config syntax with a sub checksum config 
did you use SHA-1 by yourself ?
 all image i used/see on internet are providing only SHA-256/md5 value

Reply:
I personally don't use it myself, but I don't see a reason to limit the options in the packer build from what is provided in the API. Undoubtably, there are people out there using sha1 still.

Reply:
let me see how/if we can do it without complexifying the config too much

Reply:
I already implemented a way to do it 😁

Reply:
yes i know 😁  , but i don't want an additional checksum substructure
there is already  lot of issues in the actual config format, and plenty we can't change without breaking existing config sadly